upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -31,3 +31,6 @@ There you can find links to upgrade notes for other versions too.
 - add phpstan-symfony extension ([#1961](https://github.com/shopsys/shopsys/pull/1961))
     - see #project-base-diff to update your project
 
+- introduced sniff for strict comparasion ([#1658](https://github.com/shopsys/shopsys/pull/1658))
+    - this change is optional but will be required in next major version
+    - see #project-base-diff to update your project
```

Comment:
```suggestion
- introduced sniff for strict comparasion ([#1658](https://github.com/shopsys/shopsys/pull/1658))
    - this change is temporary to help you prepare your project for next major version, where this will be required directly in coding-standards package
    - see #project-base-diff to update your project
```

Reply:
Nice!